src/pages/Dashboard/Dashboard.module.scss

```diff
@@ -2,27 +2,31 @@
 
 .Dashboard {
   @include select(PageContentMainBody) {
-    display: flex;
-    flex-direction: row;
-    flex-wrap: wrap;
+    width: 100%;
```

Comment:
For block level elements that are not absolutely positioned a width of 100% is redundant as block level elements are 100% width by default.

Reply:
OK